.github/workflows/test-aries-vcr-issuer.yml

```diff
@@ -0,0 +1,32 @@
+name: test-aries-vcr-issuer
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+defaults:
+  run:
+    shell: bash
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'schedule' && github.repository == 'hyperledger/aries-vcr-issuer-controller') || (github.event_name != 'schedule')
```

Comment:
Wouldn't this trigger the job basically on any event?
Is the repository name a future move (e.g.: donating code), or a typo?

Reply:
> Wouldn't this trigger the job basically on any event?

No - this isn't a trigger (the trigger is in the "on:" section), this just filters whether to run the tests *after* the workflow has already been triggered.

> Is the repository name a future move (e.g.: donating code), or a typo?

Yes (future move) and yes (also a typo, since it's not moved *yet*)